PhysicsTools/Heppy/python/physicsutils/EffectiveAreas.py

```diff
@@ -86,23 +86,23 @@
                        }
         },
     Fall17 = {
-        # https://github.com/cms-sw/cmssw/blob/CMSSW_9_4_X/RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_92X.txt
+        # https://github.com/cms-sw/cmssw/blob/master/RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_94X.txt
         'electron' : { '03' : 
-                       [ (1.000, 0.1566),
-                         (1.479, 0.1626),
-                         (2.000, 0.1073),
-                         (2.200, 0.0854),
-                         (2.300, 0.1051),
-                         (2.400, 0.1204),
-                         (2.500, 0.1524) ],
+                       [ (1.000, 0.1440),
```

Comment:
This is the change Fall17  v1 -> Fall 17 v2?
If yes, I think it would be best to define a new set Fall17v2 and rename the current one as Fall17v1 (so that people are forced to take a decision)

Reply:
Yes, it is Fall 17 v2. @GaelTouquet can certainly add back Fall17v1.